lib/handler/compress/brotli.cc

```diff
@@ -72,8 +72,7 @@ void brotli_context::_emit(bool is_last, bool force_flush)
     uint8_t *output;
     size_t out_size;
     bool ret = brotli_->WriteBrotliData(is_last, force_flush, &out_size, &output);
-    assert(ret);
-    if (out_size != 0)
+    if (ret && out_size != 0)
```

Comment:
Does changing this to the below solve the issue? 
```
assert(ret);
(void)ret;
if (out_size != 0)
```

The intent here is to retain the behavior as-is (we want to check the behavior in debug builds but not in release builds).

Reply:
Sure it will solve :-)

Reply:
I've seen this:
```(86 of 782): warning: ‘%u’ directive writing between 1 and 10 bytes into a region of size 6 [-Wformat-overflow=]                                                                                                                                                                                            426,39        99%
```